test/17.js

```diff
@@ -0,0 +1,26 @@
+const expect = require('chai').expect;
+let solution = require('../solutions/17').solution;
+//solution = require('../yourSolution').solution;
+
+describe('Test Case 1: 3x3', () => {
```

Comment:
It doesn't make sense to have a describe and only have 1 test case in it. Usually we split up and group tests into different describe functions when there are too many test cases.